astropy/io/fits/tests/test_table.py

```diff
@@ -3288,3 +3288,24 @@ def test_empty_table(tmpdir):
 
     with fits.open(ofile) as hdul:
         assert hdul['TEST'].data.size == 0
+
+
+def test_a3dtable(tmpdir):
+    testfile = str(tmpdir.join('test.fits'))
+    hdu = fits.BinTableHDU.from_columns([
+        fits.Column(name='FOO', format='J', array=np.arange(10))
+    ])
+    hdu.header['XTENSION'] = 'A3DTABLE'
+    hdu.writeto(testfile, output_verify='ignore')
+
+    with fits.open(testfile) as hdul:
+        assert hdul[1].header['XTENSION'] == 'A3DTABLE'
+
+        with catch_warnings() as w:
+            hdul.verify('fix')
+
+        assert str(w[0].message) == 'Verification reported errors:'
+        assert str(w[2].message)\
```

Comment:
A bit too late to the review, but I would have preferred it to be a slightly longer line than to use `\`. :woman_shrugging:

Reply:
`io.fits` usually respects the pep8 limit but yeah here it is not the most beautiful ...
I'm getting used to black with its 88 characters limit, which is quite nice, so I would be happy to use longer lines if we put a `max-line-length = 88` line in flake8's settings :wink:

Reply:
Heheh... I usually just append a `# noqa` so it goes away. :wink:

Reply:
this particular one could be done by adding a line break after `.endswith(`

p.s. Nice simple fix!

Reply:
this particular one could be done by adding a line break after `.endswith(`

p.s. Nice simple fix!

Reply:
Great idea, @mhvk ! See #9021.